src/components/ComboBox/SingleComboBox.tsx

```diff
@@ -412,17 +412,7 @@ const ActualSingleComboBox = <T,>(
   ])
 
   return (
-    <div
-      {...props}
-      {...wrapperStyleProps}
-      ref={outerRef}
-      role="combobox"
-      aria-haspopup="listbox"
-      aria-controls={listBoxId}
-      aria-expanded={isFocused}
-      aria-invalid={error || undefined}
-      onKeyPress={handleKeyPress}
-    >
+    <div {...props} {...wrapperStyleProps} ref={outerRef} role="presentation">
```

Comment:
onKeyPress が移動できたので、`role=presentation` は不要です!

```suggestion
    <div {...props} {...wrapperStyleProps} ref={outerRef}>
```